includes/class-wc-calypso-bridge-setup.php

```diff
@@ -45,6 +47,31 @@ private function __construct() {
 		add_filter( 'pre_option_woocommerce_homescreen_enabled', array( $this, 'always_enable_homescreen' ) );
 	}
 
+	/**
+	 * Initial admin panel setup (options etc.)
+	 *
+	 * This is a controlled function that should run once.
+	 * An option named `woocommerce_atomic_initialized` is used to determine pristine condition.
+	 *
+	 * @since 1.9.4
+	 * @return void
+	 */
+	public function setup_admin_one_time_operations() {
+
+		// Make sure setup runs only once.
+		if ( 'yes' === get_option( 'woocommerce_atomic_initialized', 'no' ) ) {
+			return;
+		}
+
+		// Delete all existing `Coupon Page Moved` notes from the DB.
+		if ( class_exists( 'Automattic\WooCommerce\Admin\Notes\Notes' ) ) {
+			Automattic\WooCommerce\Admin\Notes\Notes::delete_notes_with_name( 'wc-admin-coupon-page-moved' );
+		}
+
+		// Mark setup complete.
+		update_option( 'woocommerce_atomic_initialized', 'yes' );
+	}
```

Comment:
@xristos3490 I've moved the one-time operations function here. I some minor modifications, let me know if you agree (here's your PR https://github.com/Automattic/wc-calypso-bridge/pull/811/files#diff-6913cd5881364970fe5a0344296a03005ccab72b20b844d5e040e649ba91531aR59 )